src/modules/Dashboard/components/NotesModal/__tests__/NotesModal.test.jsx

```diff
@@ -0,0 +1,94 @@
+import { configure, mount } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import * as React from "react";
+import { Alert, Modal } from "react-bootstrap";
+import configureStore from "redux-mock-store";
+import { Provider } from "react-redux";
+import { cleanup } from "@testing-library/react";
+
+import NotesModal from "../NotesModal";
+
+configure({ adapter: new Adapter() });
+
+afterEach(cleanup);
+
+describe("NotesModal", () => {
+    const imageId = "123e4567-e89b-12d3-a456-426614174000";
+    const mockStore = configureStore([]);
+
+    const render = ({ initialValue, show = true, showNotesSucceededMessage = false }) => {
+        const initialState = {
+            dashboard: {
+                showNotesSucceededMessage,
+            },
+        };
+        const store = mockStore(initialState);
+
+        return mount(
+            <Provider store={store}>
+                <NotesModal
+                    handleClose={jest.fn()}
+                    show={show}
+                    imageId={imageId}
+                    initialValue={initialValue}
+                />
+            </Provider>,
+        );
+    };
+
+    /**
+     * Checks if the Modal is rendered, when the show prop is true.
+     */
+    it("Modal rendered", () => {
+        const component = render({});
+        expect(component.find(Modal).length).toEqual(1);
+    });
+
+    /**
+     * Checks if the Modal isn't rendered, when the show prop is false.
+     */
+    it("Modal doesn't rendered", () => {
+        const component = render({ show: false });
+        const modal = component.find(Modal);
+        const nestedModal = modal.children();
+        expect(nestedModal.children().length).toEqual(0);
+    });
+
+    /**
+     * Checks if the succeeded alert is displayed, when the showNotesSucceededMessage
+     * prop is true.
+     */
+    it("Succeeded alert is displayed", () => {
+        const component = render({ showNotesSucceededMessage: true });
+        expect(component.find(Alert).prop("show")).toEqual(true);
+    });
+
+    /**
+     * Checks if the succeeded alert isn't displayed, when the showNotesSucceededMessage
```

Comment:
Checks if the succeeded alert isn't shown when the showNotesSucceededMessage prop is false.